dpc-impl/config/routes.rb

```diff
@@ -24,5 +24,6 @@
   if Rails.env.development?
     require 'sidekiq/web'
     mount Sidekiq::Web, at: '/sidekiq'
+    mount LetterOpenerWeb::Engine, at: "/letter_opener" if Rails.env.development?
```

Comment:
It's already inside an `if` block 😄 
```suggestion
    mount LetterOpenerWeb::Engine, at: "/letter_opener"
```